src/MapfileStyleParser.ts

```diff
@@ -28,20 +37,331 @@ export class MapfileStyleParser implements StyleParser {
     Object.assign(this, opts);
   }
 
+  /**
+   * Get the name for the Style from the Mapfile Object. Returns the GROUP value of the LAYER
+   * if defined or the NAME value of the LAYER if defined or an empty string.
+   *
+   * @param {object} mapfileObject The Mapfile object representation (created with xml2js)
+   * @return {string} The name to be used for the GeoStyler Style Style
+   */
+  getStyleNameFromMapfileObject(mapfileObject: any): string {
```

Comment:
Could we introduce a type for mapfileObject in the parser? or is it too convoluted?

Reply:
leaving it as is for now